lib/translations/pt-BR.js

```diff
@@ -0,0 +1,56 @@
+
+
+module.exports = {
```

Comment:
**Suggestion:** The explicit call to `module.exports` is really necessary? I think that the translation file needs be a plain `.json` file instead a `.js` file